man/systemd.network.xml

```diff
@@ -859,6 +859,11 @@ Table=1234</programlisting></para>
           <citerefentry><refentrytitle>networkd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
           </para>
 
+          <para>Note, typically, enabling per-interface <varname>IPv6Forwarding=</varname> on two or more
+          interfaces does not work. If you want to enable IPv6 forwarding, enable the global setting in
+          <citerefentry><refentrytitle>networkd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>,
+          and optionally disable the per-interface setting on some interfaces.</para>
```

Comment:
From the sysctl documentation:
> IPv4 and IPv6 work differently here; e.g. netfilter must be used to control which interfaces may forward packets and which not.

So does disabling `IPv6Forwarding` per interface really work?

Reply:
> From the sysctl documentation:
> 
> > IPv4 and IPv6 work differently here; e.g. netfilter must be used to control which interfaces may forward packets and which not.
> 
> So does disabling `IPv6Forwarding` per interface really work?

Depends on the purpose. The per-interface setting is also used at several places in the kernel.

Reply:
> and optionally disable the per-interface setting on some interfaces
To me, this reads as "enable it globally but you can still use per interface setting to _disable_ forwarding per interface". And this doesn't seem to be the case, as per documentation.

Reply:
> > and optionally disable the per-interface setting on some interfaces
> 
> To me, this reads as "enable it globally but you can still use per interface setting to _disable_ forwarding per interface". And this doesn't seem to be the case, as per documentation.

Not sure. What feature you cannot disable with the per-interface setting?

Reply:
> What feature you cannot disable with the per-interface setting?

IPv6 Forwarding.

Reply:
Have you tested?

Reply:
Yes, I can set the `IPv6Forwarding` option on two interfaces, but they still forward v6 packets.